### PR TITLE
Add selection style to CSS

### PR DESCRIPTION
**Pull request by Bobbbay**
# Fixes

### Changes Summary

- Added better-looking and colored ::selection
- Works in almost every major browser ([caniuse](https://caniuse.com/#feat=css-selection))

This pull request **is** ready for review.

@lewisgoddard @btkostner @danrabbit

### CODE REVIEW

File: _styles/main.css [hunk 1]
```diff
@@ -42,6 +42,18 @@ body {
     box-sizing: inherit;
 }
 
+/* Code for old Firefox and KaiOS */
+::-moz-selection {
+    color: #fff;
+    background: #3995d3;
```
Comment (btkostner) [inline _styles/main.css]:
```suggestion
    background-color: #3995d3;
```

File: _styles/main.css [hunk 2]
```diff
@@ -42,6 +42,18 @@ body {
...
+}
+
+/* Code for modern browsers */
+::selection {
+    color: #fff;
+    background: #3995d3;
```
Comment (btkostner) [inline _styles/main.css]:
```suggestion
    background-color: #3995d3;
```

**Comment by btkostner** [comment]
Asking for a UX review on this to see if we should override the user's default.

**Comment by cassidyjames** [comment]
Eh, I'm not really inclined to override the user/OS default, but I also don't feel strongly about it.

**Comment by Bobbbay** [comment]
Oops, my bad, in so preoccupied I said background and and not background-color :(

In my opinion, ```::selection``` is ingenious and most sites should have one, too. This is mostly used on designer sites, landing pages and **OS sites**. Everyone has their own opinion, but I think it would increase the aesthetic of the website. Comments? Opposing opinions? Suggestions? I'd love to hear them!

Happy kernels, Bobbbay

**Comment by Bobbbay** [comment]
UX doesn't seem to be respondong... Would you (@btkostner) like me to contact some of my professional UX sources? They should get back to me quite quickly, if they don't have anything on their schedule. 

Happy commits, Bobbbay

**Comment by cassidyjames** [comment]
@Bobbbay I did respond, and I'm on the UX team. I don't see a strong argument to override OS and browser default styles, but I also don't really care either way.

**Comment by Bobbbay** [comment]
Sorry, my bad, I hadn't realised you were the UX representative. Clumsy me...

Well then, if UX doesn't have a strong argument against it, I think we should merge the pull!

Happy pulling, Bobbbay

**Comment by Bobbbay** [comment]
So another seperate PR? If so, should it automatically merge/fix/etc. this PR?

**Comment by danrabbit** [comment]
proposed changes for the OS selection style, so let's put this on hold until that's reviewed https://github.com/elementary/stylesheet/pull/572

**Comment by Bobbbay** [comment]
That commit is reviewed and merged, anything over here?

**Comment by btkostner** [comment]
@Bobbbay this should be updated to reflect the colors that were merged in the gtk stylesheet.

**Comment by Bobbbay** [comment]
Alright, just checking: that's rgba(100, 186, 255, 0.85), right?

**Comment by btkostner** [comment]
```css
selection {
	background-color: rgba(140, 213, 255, 0.3);
	color: rgb(0, 46, 153);
	text-shadow: none;
}
```

**Comment by Bobbbay** [comment]
Alright, looks good, and dang, I was way off 😀

**Comment by Bobbbay** [comment]
Alright, done, and woahhh, I was waaay off :) 

I think this is ready to be merged.

**Comment by btkostner** [comment]
@Bobbbay Thank you for this!

**Pull request merged**